src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -53,12 +48,28 @@ public class AccessController : Controller
         }
 
         [AllowAnonymous, HttpGet, HttpPost, IgnoreAntiforgeryToken]
-        public async Task<IActionResult> Authorize([ModelBinder(typeof(OpenIddictMvcBinder))] OpenIdConnectRequest request)
+        public async Task<IActionResult> Authorize()
         {
+            var response = HttpContext.GetOpenIddictServerResponse();
```

Comment:
These additional are moved from the `[OpenIdControllerAttribute]` that I'd like to remove from RTM as it doesn't do what people think it does and shouldn't be used in any user-defined controller.